sig-scalability/annual-report-2022.md

```diff
@@ -24,58 +24,59 @@
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Each subproject could benefit from additional hands. However, the Scalability Test Frameworks and Scalability and
+     Performance tests and validation are the ones where we can grow contributors - the other require both very deep and
+     wide understanding of Kubernetes before making reasonable contributions.
```

Comment:
Is there a repo or issue or something you can link out to for this?

Reply:
Yes - it's this repo and most of issues are good one to start with:
https://github.com/kubernetes/perf-tests/issues

Will add.

Reply:
This is a more recent development - https://github.com/kubernetes/test-infra/issues/29139.
But I think we should explicitly call out the need to decentralize/load-balance scale testing for the project's sustainability. Wojtek already mentioned about the test framework above, but we also need more help beyond that imo - like chasing regressions, improving test health, etc.

Reply:
This is super important, but this is 2023. The report is only for 2022. It will appear in the report next year.